framework/core/command_buffer.cpp

```diff
@@ -277,7 +281,7 @@ void CommandBuffer::bind_index_buffer(const core::Buffer &buffer, VkDeviceSize o
 	vkCmdBindIndexBuffer(get_handle(), buffer.get_handle(), offset, index_type);
 }
 
-void CommandBuffer::bind_lighting(LightingState &lighting_state, uint32_t set, uint32_t binding)
+void CommandBuffer::bind_lighting(vkb::rendering::LightingStateC &lighting_state, uint32_t set, uint32_t binding)
```

Comment:
we're already in the vkb namespace.  It's safe to use rendering::LightingStateC

Reply:
I tend to use the complete namespace. I think, it's easier to mentally locate the used entity.
If you want, I can remove that `vbk::` here, though.